### ISSUE TITLE
[Bug]: NOTE: Process `pipeline:assembleCore (1)` terminated with an error exit status (1) -- Execution is retried (1)

### ISSUE THREAD

**Issue by nathanrussart** [epi2me-labs/wf-clone-validation#7]
### What happened?

Hello,

Trying to run the clone-validation workflow and it keeps terminating during assembleCore.  Any help is appreciated!

### Operating System

Windows 10

### Workflow Execution

EPI2ME Labs desktop application

### Workflow Execution - EPI2ME Labs Versions

3.1.2

### Workflow Version

v.0.1.9

### Relevant log output

```shell
N E X T F L O W ~ version 21.10.6

Launching `epi2me-labs/wf-clone-validation` [hopeful_becquerel] - revision: db0bf2a6a4 [v0.1.9]

NOTE: Your local project version looks outdated - a different revision is available in the remote repository [6660d8c662]

Core Nextflow options

revision : v0.1.9

runName : hopeful_becquerel

containerEngine: docker

launchDir : /mnt/c/Users/lab.rd/epi2melabs-data/nextflow

workDir : /mnt/c/Users/lab.rd/epi2melabs-data/nextflow/instances/2022-07-27-08-40_wf-clone-validation_o956T2Y9g2LWrsc3rnBegb/work

projectDir : /home/aldevronrd/.nextflow/assets/epi2me-labs/wf-clone-validation

userName : aldevronrd

profile : standard

configFiles : /home/aldevronrd/.nextflow/assets/epi2me-labs/wf-clone-validation/nextflow.config

Input/output options

fastq : /mnt/c/Users/lab.rd/epi2melabs-data/plasmid_verification_25july2022/plasmids/20220725_1542_MN38783_alr150_7c41a874/high_accuracy

sample_sheet : /mnt/c/Users/lab.rd/Desktop/Clone Validation Workflow files/sample_sheet.csv

out_dir : /mnt/c/Users/lab.rd/epi2melabs-data/nextflow/instances/2022-07-27-08-40_wf-clone-validation_o956T2Y9g2LWrsc3rnBegb/output

primers : /home/aldevronrd/.nextflow/assets/epi2me-labs/wf-clone-validation/data/primers.tsv

Reference genome options

db_directory : /mnt/c/Users/lab.rd/wf-clone-validation-db.tar

host_reference : /mnt/c/Users/lab.rd/epi2melabs-data/nextflow/NO_HOST_REF

!! Only displaying parameters that differ from the pipeline defaults !!

------------------------------------------------------

If you use wf-clone-validation for your analysis please cite:

* The nf-core framework

https://doi.org/10.1038/s41587-020-0439-x

Checking fastq input.

Non barcoded directories detected.

Warning: `--sample_sheet` given but no barcode directories found.

[eb/579fd7] Submitted process > pipeline:combineFastq (2)

[cd/10050c] Submitted process > pipeline:combineFastq (1)

[85/73d71e] Submitted process > pipeline:getParams

[7e/43715a] Submitted process > pipeline:getVersions

[df/96fe8b] Submitted process > start_ping:pingMessage (1)

[f5/891542] Submitted process > end_ping:pingMessage

[1f/154c69] Submitted process > pipeline:assembleCore (1)

[1f/154c69] NOTE: Process `pipeline:assembleCore (1)` terminated with an error exit status (1) -- Execution is retried (1)

[08/8ff59f] Submitted process > pipeline:assembleCore (2)

[08/8ff59f] NOTE: Process `pipeline:assembleCore (2)` terminated with an error exit status (1) -- Execution is retried (1)

[1b/e99812] Re-submitted process > pipeline:assembleCore (1)

[1b/e99812] NOTE: Process `pipeline:assembleCore (1)` terminated with an error exit status (1) -- Execution is retried (2)

[f4/aa9328] Re-submitted process > pipeline:assembleCore (2)

[f4/aa9328] NOTE: Process `pipeline:assembleCore (2)` terminated with an error exit status (1) -- Execution is retried (2)

[1c/51de14] Re-submitted process > pipeline:assembleCore (1)

[1c/51de14] NOTE: Process `pipeline:assembleCore (1)` terminated with an error exit status (1) -- Execution is retried (3)

[3c/8e9360] Re-submitted process > pipeline:assembleCore (2)

[3c/8e9360] NOTE: Process `pipeline:assembleCore (2)` terminated with an error exit status (1) -- Execution is retried (3)

[a2/93b468] Re-submitted process > pipeline:assembleCore (1)

[a2/93b468] NOTE: Process `pipeline:assembleCore (1)` terminated with an error exit status (1) -- Execution is retried (4)

[4d/6e2e0e] Re-submitted process > pipeline:assembleCore (2)

[4d/6e2e0e] NOTE: Process `pipeline:assembleCore (2)` terminated with an error exit status (1) -- Execution is retried (4)

[d5/3ffa90] Re-submitted process > pipeline:assembleCore (1)

[d5/3ffa90] NOTE: Process `pipeline:assembleCore (1)` terminated with an error exit status (1) -- Execution is retried (5)

[9f/1889d1] Re-submitted process > pipeline:assembleCore (2)

[9f/1889d1] NOTE: Process `pipeline:assembleCore (2)` terminated with an error exit status (1) -- Execution is retried (5)

[cb/ca14bb] Re-submitted process > pipeline:assembleCore (1)

[75/49461e] Re-submitted process > pipeline:assembleCore (2)

[8e/fe55a6] Submitted process > pipeline:downsampledStats (1)

[77/88edf5] Submitted process > pipeline:downsampledStats (2)

[e3/831aec] Submitted process > pipeline:inserts

[f1/f6ba48] Submitted process > pipeline:runPlannotate

Error executing process > 'pipeline:runPlannotate'

Caused by:

Missing output file(s) `*annotations.bed` expected by process `pipeline:runPlannotate`

Command executed:

if [ -e "assemblies/OPTIONAL_FILE" ]; then

assemblies=""

else

assemblies="--sequences assemblies/"

fi

run_plannotate.py $assemblies --database wf-clone-validation-db.tar

Command exit status:

0

Command output:

(empty)

Command error:

/home/epi2melabs/conda/lib/python3.6/site-packages/blinker/base.py:110: DeprecationWarning: invalid escape sequence \*

"""

/home/epi2melabs/conda/lib/python3.6/site-packages/blinker/base.py:180: DeprecationWarning: invalid escape sequence \*

"""

/home/epi2melabs/conda/lib/python3.6/site-packages/blinker/base.py:252: DeprecationWarning: invalid escape sequence \*

"""

/home/epi2melabs/conda/lib/python3.6/site-packages/streamlit/config_option.py:146: DeprecationWarning: invalid escape sequence \_

"\_?[a-z][a-z0-9]*"

/home/epi2melabs/conda/lib/python3.6/site-packages/streamlit/config_option.py:149: DeprecationWarning: invalid escape sequence \.

"\."

/home/epi2melabs/conda/lib/python3.6/site-packages/streamlit/code_util.py:83: DeprecationWarning: invalid escape sequence \]

inputs = re.split(",\\s*(?![^(){}[\]]*\\))", line_args)

/home/epi2melabs/conda/lib/python3.6/site-packages/validators/url.py:74: DeprecationWarning: invalid escape sequence \.

u"(?:\.(?:[a-z\u00a1-\uffff0-9]-?)*[a-z\u00a1-\uffff0-9]+)*"

/home/epi2melabs/conda/lib/python3.6/site-packages/validators/url.py:76: DeprecationWarning: invalid escape sequence \.

u"(?:\.(?:[a-z\u00a1-\uffff]{2,}))"

/home/epi2melabs/conda/lib/python3.6/site-packages/plannotate/pLannotate_app.py:244: DeprecationWarning: invalid escape sequence \*

st.write("\*plasmid is displayed as cirucular, though pLannotate is treating this as a linear construct")

Work dir:

/mnt/c/Users/lab.rd/epi2melabs-data/nextflow/instances/2022-07-27-08-40_wf-clone-validation_o956T2Y9g2LWrsc3rnBegb/work/f1/f6ba480c33c8b8f63d60047194bdd1

Tip: you can replicate the issue by changing to the process work dir and entering the command `bash .command.run`

WARN: To render the execution DAG in the required format it is required to install Graphviz -- See http://www.graphviz.org for more info.
```

**Comment by sarahjeeeze** [comment]
Hi Thanks for pointing this out, I have released a new version 0.2.2 that should fix this that will be available if you update epi2melabs to the latest version.

**Comment by sarahjeeeze** [comment]
Closing through lack of response from original poster.
